src/mgmt/tests/mocks.h

```diff
@@ -95,4 +95,15 @@ void mock_ds_mgmt_list_pools_setup(void);
 void mock_ds_mgmt_list_pools_teardown(void);
 void mock_ds_mgmt_list_pools_gen_pools(size_t num_pools);
 
+/*
+ * Mock ds_mgmt_pool_list_cont
+ */
+extern int				 ds_mgmt_pool_list_cont_return;
+extern struct daos_pool_cont_info	*ds_mgmt_pool_list_cont_out;
+extern uint64_t				 ds_mgmt_pool_list_cont_nc_out;
```

Comment:
Nit - alignment? Or is this Github's display error?

Reply:
github display error